src/ast/CMakeLists.txt

```
@@ -98,5 +90,6 @@ else()
     # have libLLVM.so, drop `USE_SHARED` to avoid forcing link to `-lLLVM`.
     llvm_config(ast bpfcodegen ipo irreader mcjit orcjit)
   endif()
-  target_link_libraries(ast PUBLIC libclang clangDriver clangFrontend clangCodeGen)
+
+  target_link_libraries(ast PUBLIC libclang clang-cpp)
```

Nit: I think that we could do with
```suggestion
  target_link_libraries(ast PUBLIC clang clang-cpp)
```

---

This doesn't work as expected, I believe because this is the cmake target name not `find_library` name.